tarodb/asset_minting_test.go

```diff
@@ -76,6 +74,77 @@ func assertSeedlingBatchLen(t *testing.T, batches []*tarogarden.MintingBatch,
 	require.Len(t, batches[0].Seedlings, numSeedlings)
 }
 
+// storeGroupGenesis generates a group genesis asset and inserts it into the DB.
+// The group genesis asset information needs to be in the DB before reissuance.
+func storeGroupGenesis(t *testing.T, ctx context.Context,
+	store *AssetMintingStore, assetType asset.Type) (uint64,
+	*btcec.PrivateKey, *asset.AssetGroup) {
+
+	initialGen := asset.RandGenesis(t, assetType)
+	privDesc, groupPriv := randKeyDesc(t)
+
+	// Generate the signature for our group genesis asset.
+	genSigner := asset.NewRawKeyGenesisSigner(groupPriv)
+	groupKey, err := asset.DeriveGroupKey(
+		genSigner, privDesc, initialGen, nil,
+	)
+	require.NoError(t, err)
+
+	initialAsset := asset.RandAssetWithValues(
+		t, initialGen, groupKey, asset.RandScriptKey(t),
+	)
+
+	// Insert the group genesis asset, which will also insert the group key
+	// and genesis info needed for reissuance.
+	var writeTxOpts AssetStoreTxOptions
+	dbErr := store.db.ExecTx(ctx, &writeTxOpts,
+		func(q PendingAssetStore) error {
```

Comment:
Style nit: could use a closure above to make the line breaking a bit more uniform here.